assets/scss/partials/components/_notice.scss

```diff
@@ -0,0 +1,51 @@
+.notice {
+  padding-left: 0;
+  padding-right: 0;
+  width: 100%;
+  @include themed() {
+    background-color: t('notice-info');
+  }
+
+  &--update {
+    @include themed() {
+      background-color: t('notice-update');
+    }
+  }
+  &--warning {
+    @include themed() {
+      background-color: t('notice-warning');
+    }
+  }
+
+  &__title {
+    background-color: rgb(96, 134, 180);
+    float: right;
+    font-weight: 300;
+    letter-spacing: 0.025em;
+    margin-bottom: 0.25rem;
+    margin-left: 1rem;
```

Comment:
I don' think we need any `margin` here when you control spacing with the `padding` of the notice's content.